k8s-deploy/kubefate.go

```diff
@@ -27,12 +27,13 @@ import (
 
 func main() {
 	config.InitViper()
-	logging.InitLog()
 	err := viper.ReadInConfig()
 	if err != nil {
 		log.Debug().Err(err).Msg("load config.yaml error")
```

Comment:
Could you please also replace this line with `fmt.Println()` ?

Reply:
@jiahaoc1993 replace `log.Debug().Err(err).Msg("load config.yaml error")` with `fmt.Println("load config.yaml error")`?

Reply:
@jiahaoc1993 I have been replace `log` with `fmt.Println`, is this what you mean?

Reply:
Looks good to me now. @asdfsx could you please sign-off this commit? Ref: https://github.com/FederatedAI/KubeFATE/pull/369/checks?check_run_id=2083856881